node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -75,7 +75,7 @@ const (
 	IscsiHostRexExPath          = "/sys/class/iscsi_host/host*/device/session*/iscsi_session/session*/targetname"
 	sysDeviceSymLinkFormat      = "/sys/block/%s/device"
 	sysDeviceDeletePathFormat   = sysDeviceSymLinkFormat + "/delete"
-	blockDevCmd                 = "blockdev"
+	BlockDevCmd                 = "blockdev"
 	flushBufsFlag               = "--flushbufs"
 	mpathdSeparator             = ","
 	multipathdCmd               = "multipathd"
```

Comment:
it seems `multipathdCmd` is duplicated, I suggest we do the same for `blockdevCmd`, at least until we find a common place for these

Reply:
done

Reply:
I still see a diff in this file

Reply:
done now